modules/disk-uploader/pkg/image/image.go

```diff
@@ -16,7 +17,26 @@ import (
 	tar "kubevirt.io/containerdisks/pkg/build"
 )
 
-func Build(diskPath string) (v1.Image, error) {
+var labelEnvPairs = []struct {
+	label, env string
+}{
+	{"instancetype.kubevirt.io/default-instancetype", "INSTANCETYPE_KUBEVIRT_IO_DEFAULT_INSTANCETYPE"},
```

Comment:
I'd use the simpler map[string]string and would iterate over it down below.

Reply:
Yep that would be better, but I don't see it as key:value here, IMO both values are keys (key:key).